### ISSUE TITLE
Default gem install - openssl, can't activate

### ISSUE THREAD

**Issue by MSP-Greg** [rubygems/rubygems#2699]
Working on Travis, there is no reasonable way to test against OpenSSL 1.1.1.  Using macOS, 1.1.1 is available, but Ruby 2.5.x & 2.6.x are compiled with OpenSSL 1.1.0.

So, I thought I'd install the OpenSSL gem, which is currently the same version as the default gem.

How do I activate it?

**EDIT:** I also installed the gem in Ruby 2.4.x, which has an older version of Ruby OpenSSL.  It activated correctly.

This issue is related to:

  - [ ] Network problems
  - [ ] Installing a library
  - [ ] Publishing a library
  - [X] The command line `gem`
  - [ ] Other

Here are my current environment details:

```
Travis macOS Ruby 2.5 & 2.6
```

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

**Comment by deivid-rodriguez** [comment]
Can you help us reproduce this? I tried activating the latest and older versions of the gem on my machine and it worked fine.

**Comment by MSP-Greg** [comment]
@deivid-rodriguez 

It needs to be the same version.  Both 2.5.5 & 2.6.2 have a default gem 2.1.2, built with 1.1.0.  I want to activate the current release, also 2.1.2, but built with OpenSSL 1.1.1.  I checked by looking at:

```ruby
OpenSSL::Buffering.instance_method(:close).source_location
```
and the location was the default, not the installed gem...

**Comment by deivid-rodriguez** [comment]
Ah, I see. Maybe https://github.com/rubygems/rubygems/pull/2112 would fix this?

**Comment by deivid-rodriguez** [comment]
Closed by #2112.
